### PR TITLE
[Fix/doxygen-tag] Fix overwriting result bug

### PR DESCRIPTION
**Pull request by ohsewon**
**Changelog:**

Version 2:
- Add skip status for none source code commit.

Version 1:
- 'success' value is default. so set value to 'success' is not necessary.

Signed-off-by: sewon.oh <sewon.oh@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [*]Passed [ ]Failed []Skipped

### CODE REVIEW

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 1]
```diff
@@ -163,12 +163,10 @@ function pr-format-doxygen-tag(){
                         break
                     else
                         echo "[DEBUG] $doxygen_lang: passed. file name: $curr_file, ($doxygen_rule_num)/$doxygen_rule_all tags are found."
-                        check_result="success"
                     fi
                     ;;
                 * )
                     echo "[DEBUG] ( $curr_file ) file is not source code with the text format."
-                    check_result="success"
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: after removing this line, let's introduce a "skip" condition to handle not source codes.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@ function pr-format-doxygen-tag(){
     echo "[MODULE] TAOS/pr-format-doxygen-tag: Check if source code includes required doxygen tags for doxygen documentation."
     # Inspect all *.patch files that are fetched from a commit file
     FILELIST=`git show --pretty="format:" --name-only --diff-filter=AMRC`
-    check_result="success"
+    check_result="skip"
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: Good job.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 3]
```diff
@@ -53,6 +53,11 @@ function pr-format-doxygen-tag(){
                     # Append a doxgen rule step by step
                     doxygen_basic_rules="@file @brief" # @file and @brief to inspect file
                     doxygen_advanced_rules="@author @bug" # @author, @bug to inspect file, @brief for to inspect function
+
+                    # When check_result is skip, set value to success(default)
+                    if [[ $check_result == "skip" ]]; then
+                        check_result="success"
+                    else
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: It's strange. else or fi? please check the statement correct?

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 4]
```diff
@@ -178,6 +187,10 @@ function pr-format-doxygen-tag(){
         echo "[DEBUG] Passed. doxygen documentation."
         message="Successfully source code(s) includes doxygen document correctly."
         cibot_pr_report $TOKEN "success" "TAOS/pr-format-doxygen-tag" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "${GITHUB_WEBHOOK_API}/statuses/$input_commit"
+    elif [[ $check_result == "skip" ]]; then
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: Looks good.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 5]
```diff
@@ -149,6 +154,12 @@ function pr-format-doxygen-tag(){
                     doxygen_rules="@package @brief"
                     doxygen_rule_num=0
                     doxygen_rule_all=0
+
+                    # When check_result is skip, set value to success(default)
+                    if [[ $check_result == "skip" ]]; then
+                        check_result="success"
+                    else
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: s/else/fi/? Please check it.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh (hunk 3 above)
Comment (ohsewon) [reply ci/taos/plugins-good/pr-format-doxygen-tag.sh]: fixed. thanks.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh (hunk 5 above)
Comment (ohsewon) [reply ci/taos/plugins-good/pr-format-doxygen-tag.sh]: fixed. thanks.

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 6]
```diff
@@ -53,6 +53,11 @@ function pr-format-doxygen-tag(){
                     # Append a doxgen rule step by step
                     doxygen_basic_rules="@file @brief" # @file and @brief to inspect file
                     doxygen_advanced_rules="@author @bug" # @author, @bug to inspect file, @brief for to inspect function
+
+                    # When check_result is skip, set value to success(default)
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: Please update the statement. 


`# When a current file is a source code, change a default value of $check_result from 'skip' to 'success'.`

File: ci/taos/plugins-good/pr-format-doxygen-tag.sh [hunk 7]
```diff
@@ -149,6 +154,12 @@ function pr-format-doxygen-tag(){
                     doxygen_rules="@package @brief"
                     doxygen_rule_num=0
                     doxygen_rule_all=0
+
+                    # When check_result is skip, set value to success(default)
```
Comment (leemgs) [inline ci/taos/plugins-good/pr-format-doxygen-tag.sh]: Please update the statement. 


`# When a current file is a source code, change a default value of $check_result from 'skip' to 'success'.`

**Pull request merged**